File: 2022-06-craft-conf-languages/examples/fibonacci.go
package main

import "fmt"

func fib(x int) int {
	a := 0
	b := 1
	for i := 0; i < x; i++ {
		next := a + b
		a = b
		b = next
	}
	return a
}

func main() {
	fmt.Println(fib(10))
}

File: 2022-06-craft-conf-languages/examples/fibonacci.cc
long fib(long x) {
    long a = 0;
    long b = 1;
    for (long i = 0; i < x; i++) {
        long next = a + b;
        a = b;
        b = next;
    }
    return a;
}

#include <iostream>

int main(void) {
    std::cout << fib(10) << std::endl;
}
